dask_geopandas/dataframe.py

```diff
@@ -0,0 +1,103 @@
+from functools import partialmethod
+import numpy as np
+import dask
+import dask.dataframe as dd
+from dask.utils import M, OperatorMethodMixin, derived_from
+
+import geopandas
+
+
+class _Frame(dd.core._Frame, OperatorMethodMixin):
+    """ Superclass for DataFrame and Series
+
+    Parameters
+    ----------
+    dsk : dict
+        The dask graph to compute this DataFrame
+    name : str
+        The key prefix that specifies which keys in the dask comprise this
+        particular DataFrame / Series
+    meta : geopandas.GeoDataFrame, geopandas.GeoSeries
+        An empty cudf object with names, dtypes, and indices matching the
+        expected output.
+    divisions : tuple of index values
+        Values along which we partition our blocks on the index
+    """
+    def __repr__(self):
+        s = "<dask_geopandas.%s | %d tasks | %d npartitions>"
+        return s % (type(self).__name__, len(self.dask), self.npartitions)
+
+    def to_dask_dataframe(self):
+        """Create a dask.dataframe object from a dask_geopandas object"""
+        return self.map_partitions(M.to_pandas)
+
+    @property
+    @derived_from(geopandas.base.GeoPandasBase)
+    def bounds(self):
+        return self.map_partitions(
+            getattr,
+            "bounds",
+            token=self._name + "-bounds",
+            meta=self._meta.bounds
+        )
```

Comment:
Is there a better way of putting properties onto the new classes?

Reply:
@TomAugspurger @mrocklin

Reply:
Basically I was just starting to write a `_bind_property` classmethod and wondering if such a thing already existed. I'm still working out how to set a property without knowing the name ahead of time, but I'm sure it's possible.

Reply:
Not identical, but `dask.dataframe.Accessor._delegate_property_get` may provide inspiration: https://github.com/dask/dask/blob/632f68c04154915da84c0e0befba1391a583f596/dask/dataframe/accessor.py#L16-L44

Reply:
Hmm. That is helpful, but since the property in this case is on the class itself, if we try to catch `getattr` on the fly, we get circular references. Ok I will try to see if I can wed this approach with the bind approach.

Reply:
Ok I think I've written a reasonable generic version of a property binder. Now I just need to choose whether to pass a list of properties to add or whether to bind all the properties that exist. The list approach is safer.